From 92d3b4d696d348cd6861e329f4f02ecfd14a2995 Mon Sep 17 00:00:00 2001
From: Yongjun Zhang <yzhang@cloudera.com>
Date: Thu, 14 Jan 2016 12:36:36 -0800
Subject: [PATCH 1096/2848] HDFS-9648. TestStartup.testImageChecksum is broken
 by HDFS-9569's message change. (Wei-Chiu Chuang
 via Yongjun Zhang)

(cherry picked from commit 817cc1f02a60ef4e372171415058fdc76c0d2e39)
(cherry picked from commit cfeecf113b8ee2255138c5da499b651d2fdd3251)

Change-Id: I8c14ca37a8c609c440357fa28a7865ac4437eb83
---
 .../hadoop/hdfs/server/namenode/TestStartup.java   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestStartup.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestStartup.java
index d845531..08548cc 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestStartup.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestStartup.java
@@ -543,7 +543,7 @@ private void testImageChecksum(boolean compress) throws Exception {
           fail("Should not have successfully started with corrupt image");
         } catch (IOException ioe) {
           GenericTestUtils.assertExceptionContains(
-              "Failed to load an FSImage file!", ioe);
+              "Failed to load FSImage file", ioe);
           int md5failures = appender.countExceptionsWithMessage(
               " is corrupt with MD5 checksum of ");
           // Two namedirs, so should have seen two failures
-- 
1.7.9.5

